recipe/meta.yaml

```
@@ -1,42 +1,54 @@
 {% set name = "zc.lockfile" %}
-{% set version = "2.0" %}
+{% set version = "3.0.post1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 307ad78227e48be260e64896ec8886edc7eae22d8ec53e4d528ab5537a83203b
+  url: https://github.com/zopefoundation/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: dd1394cdbf92658ddaaabc4e005fd4bdd88bff0d564e020da6e85216bf89756b
 
 build:
-  preserve_egg_dir: True
   number: 0
-  noarch: python
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-
+    - wheel
+    - setuptools
   run:
     - python
-    - setuptools
 
 test:
   imports:
     - zc
     - zc.lockfile
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: https://github.com/zopefoundation/zc.lockfile
+  summary: Basic inter-process locks
   license: ZPL 2.1
```

```suggestion
  license: ZPL-2.1
```
Match short identifier: https://spdx.org/licenses/ZPL-2.1.html